include/shards/shards.h

```diff
@@ -977,11 +1034,18 @@ typedef const struct SHObjectInfo *(__cdecl *SHFindObjectInfo)(int32_t vendorId,
 typedef int64_t(__cdecl *SHFindObjectTypeId)(SHStringWithLen name);
 typedef SHString(__cdecl *SHType2Name)(SH_ENUM_DECL SHType type);
 
+typedef void(__cdecl *SHStringGrow)(SHStringPayload *str, uint32_t newCap);
+typedef void(__cdecl *SHStringFree)(SHStringPayload *str);
+
 typedef struct _SHCore {
   // Aligned allocator
   SHAlloc alloc;
   SHFree free;
 
+  // String interface
+  SHStringGrow stringGrow;
```

Comment:
We should deprecate SHString, if this uses SHString

Reply:
Doesn't use SHString